src/domain/valueObject/serviceVersion.go

```diff
@@ -21,8 +21,7 @@ func NewServiceVersion(value interface{}) (ServiceVersion, error) {
 	stringValue = strings.TrimSpace(stringValue)
```

Comment:
InterfaceToString already has a TrimSpace, so isn't this one useless?

Reply:
This correction was made in the **VO**s refactoring branch. But I'll leave it changed here.